# Grasp pose on camera coordinate
geometry_msgs/Pose pose
# Grasp score
float64 score
# Contact point on camera coodinate
geometry_msgs/Point32 contact_point
# Instance id
float64 id